source/Calamari/Variables/VariablesFactory.cs

```diff
@@ -13,66 +15,116 @@ namespace Calamari.Variables
 {
     public class VariablesFactory
```

Comment:
Refactored this `Create` method into individual methods and cleaned up the code a bit